crates/ra_cargo_watch/src/lib.rs

```diff
@@ -316,6 +316,8 @@ impl WatchThread {
             args.push("--all-targets".to_string());
         }
         args.extend(options.args.iter().cloned());
+        args.sort();
+        args.dedup();
```

Comment:
I don't think we should dedup flags. Dedupe flag is an error, it should cause a failure.

Reply:
So do we remove the deduplication at all?